www/components/shelf/shelf.js

```diff
@@ -32,13 +30,21 @@ class shelf extends LitElement {
   }
 
   expandRoute(path) {
+    // remove / if at the end of the path in order to match our shelf paths
+    const lastPathCharacter = path.length - 1;
+
+    if (path.lastIndexOf('/') === lastPathCharacter) {
+      path = path.substring(0, lastPathCharacter);
```

Comment:
actually, is there any way to do something like this instead of the `lastPathCharacter` check?
```
    let routeShelfListIndex = this.shelfList.findIndex(list => {
      return list.path.indexOf(path) >= 0;
    });
```